streamingpro-mlsql/src/main/java/streaming/dsl/mmlib/algs/SQLDiscretizer.scala

```diff
@@ -87,7 +87,16 @@ class SQLDiscretizer(override val uid: String) extends SQLAlg with Functions wit
     return metas
   }
 
+  def isGroupParams(params: Map[String, String]): Boolean = {
```

Comment:
It's better to rename to something like `checkWithoutGroupParams` and put the require/assert in this function.
So we can use this function to make sure there no groupParam prefix in params.